modules/nw-ingress-route-secret-load-external-cert.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * ingress/routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nnw-ingress-route-secret-load-external-cert_{context}"]
+= Route Secret load For External Certificate
+
+Currently, users of OpenShift cannot easily integrate OpenShift Routes with third-party certificate management solutions like cert-manager. The existing design of the Routes API mandates that certificate data be embedded within the Route object, rather than referenced. This creates challenges for seamless integration and automated lifecycle management.
+
+This enhancement enables a seamless integration between OpenShift Routes and third-party certificate management solutions such as cert-manager. By extending the Route API, users can reference externally managed certificates via secrets, eliminating the need for manual certificate management and reducing errors. The enhancement facilitates a smoother rollout of certificate updates, ensuring the OpenShift router serves renewed certificates promptly.
```

Comment:
Which enhancement are we referring to here? Is it necessary for the end-user to know about the `o/enhancement` proposal?